### ISSUE TITLE
Roadmap for Chapter 5.3

### ISSUE THREAD

**Issue by 82817** [mschermann/data_viz_reader#1478]
Since this section started to talk about general ethic guideline for data collection, we can possibly extent this to data cleaning, and data analysis.

There are a lot of unethical ways to present data analysis in visualizations. For example, people can take out important outliers just to smooth out the graph, or they can omit certain parts of the variable to hide information that goes against their argument. 

In data cleaning, the creator can easily manipulate variables to present information in a bias way. I believe by adding these two sections, we can have a more comprehensive coverage of ethics!

**Comment by AjayDeshmukh14** [comment]
I agree with you, this section should include subsections about guidelines for data analysis and design. I found an article for the same:
https://visual.ly/blog/a-code-of-ethics-for-data-visualization-professionals/
